deeppavlov/core/agent_v2/state_schema.py

```diff
@@ -14,9 +14,14 @@ def to_dict(self):
                 'personality': self.personality}
 
 
+class Bot(User):
+    user_type = StringField(required=True, choices=['bot'], default='bot')
```

Comment:
no need in `choices`

Reply:
`to_dict` should be moved from `User` to `Bot`, if we create a separate `Bot` class.

Reply:
Removed the field altogether